src/App.tsx

```diff
@@ -7,14 +7,23 @@ import { useTldrawSettings } from "./hooks/useTldrawSettings";
 
 function App() {
   useJwtHandler();
```

Comment:
the useJwtHandler does not verify if the token is valid.
Might be out of scope for this ticket but please fix that.
Maybe I'm missing the validation.

Reply:
it's not supposed to, it just detects if your token was removed and redirects you to login page.
And it will be removed if getting your data, so calling userController/me, returns 401 or establishing connection with websocket server gives you 4401.

Reply:
ok, that makes sense.